charts/pulsar/templates/broker-statefulset.yaml

```
@@ -169,6 +169,7 @@ spec:
           echo "OK" > status;
           {{- include "pulsar.broker.zookeeper.tls.settings" . | nindent 10 }}
           bin/pulsar zookeeper-shell -server {{ template "pulsar.zookeeper.connect" . }} get {{ template "pulsar.broker.znode" . }};
+          cp /logconf/log4j2.yaml /sn-platform/conf;
```

```suggestion
          cp /logconf/log4j2.yaml {{ template "pulsar.home" . }}/conf;
```

Please use a template if possible.

---

Good idea